src/commands/cmd_bit.cc

```diff
@@ -157,7 +157,7 @@ class CommandBitPos : public Commander {
     } else if (args[2] == "1") {
       bit_ = true;
     } else {
-      return {Status::RedisParseErr, "bit should be 0 or 1"};
```

Comment:
Seems that Redis would first check if it's integer ( via `getLongFromObjectOrReply`) and then check it's `1` or `0`

```c
    /* Parse the bit argument to understand what we are looking for, set
     * or clear bits. */
    if (getLongFromObjectOrReply(c,c->argv[2],&bit,NULL) != C_OK)
        return;
    if (bit != 0 && bit != 1) {
        addReplyError(c, "The bit argument must be 1 or 0.");
        return;
    }
```

Can we keep the same?

Reply:
done